### PR TITLE
Add LightLink network

### PR DESCRIPTION
**Pull request by danatpellar**
LightLink is an EVM compatible optimistic rollup layer 2 chain featuring a gasless Enterprise Mode.

### CODE REVIEW

**Comment by mateuszradomski** [comment]
Hi, please fix the CI checks. Look at `yarn lint:fix`, `yarn format:fix` and verify that everything builds with `yarn build`.

P.S. Do not close this PR, just push changes to this one.

**Comment by lucadonnoh** [comment]
image
could you update the icon so it fits the square?

**Comment by danatpellar** [comment]
have done so @lucadonnoh, thanks for suggesting

**Review by mateuszradomski — COMMENTED** [review]
Overall, LGTM.

The structure is fine, we still need to validate the data/references/descriptions you've submitted. @lucadonnoh will probably take a peek in some free time and make the final decision.

File: packages/config/src/layer2s/lightlink.ts [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+  subtractOne,
+} from './common'
+import { getStage } from './common/stages/getStage'
+import { Layer2 } from './types'
+
+const discovery = new ProjectDiscovery('lightlink')
+
+export const lightlink: Layer2 = {
+  type: 'layer2',
+  id: ProjectId('lightlink'), // This is a simple identifier, you can use a UUID or another unique string if preferred.
```
[the remark this answers is not in the document]
Comment (Sledro) [reply packages/config/src/layer2s/lightlink.ts]: Done, thanks.

**Comment by lucadonnoh** [comment]
hey @danatpellar, i was wondering if you have any documentation regarding the in progress fraud proof system!

**Comment by lucadonnoh** [comment]
will reopen when the convo resumes

**Comment by Sledro** [comment]
hey @lucadonnoh thanks for the review!

We have recently published the latest version of our docs which contains a lot more detail:

https://docs.lightlink.io/lightlink-protocol/achitecture-and-design/protocol-architecture

**Comment by lucadonnoh** [comment]
i see that most of the components are still in progress, and as of right now there's absolutely zero validation of the DA that is posted offchain (on IPFS?).  Moreover, it doesn't seem that the data that is published onchain is in any way connected to the bridge. In the same way we didn't decide to list Blast yet among the active projects, we will list this one when some further progress is made.

**Comment by Sledro** [comment]
> i see that most of the components are still in progress, and as of right now there's absolutely zero validation of the DA that is posted offchain (on IPFS?). Moreover, it doesn't seem that the data that is published onchain is in any way connected to the bridge. In the same way we didn't decide to list Blast yet among the active projects, we will list this one when some further progress is made.

Hey @lucadonnoh - that's fair; we can respect that. 

Just to let you know, we are close to having our data submitted to Celestia on testnet. DA will be verified via our new `LLRollup.sol` contract through blobstream for every RBlock. We are also exploring posting to Greenfield as a backup solution. Do you think that once we complete the Celestia DA component, we can get listed on L2Beat with the other components marked as in development?

P.S. We saw your thread on X regarding Manta and DA. We agree, which is why we are using blobstream :)

**Comment by lucadonnoh** [comment]
yes definitely! please ping us when that’s case

**Pull request closed without merging**